chemclipse/plugins/org.eclipse.chemclipse.msd.model/src/org/eclipse/chemclipse/msd/model/support/CombinedMassSpectrumCalculator.java

```diff
@@ -96,7 +96,7 @@ public ICombinedMassSpectrum createMassSpectrum(CalculationType calculationType)
 		for(Double ion : combinedMassSpectrum.keySet()) {
 			float intensity = (float)getAbundance(ion, calculationType);
 			if(intensity > IIon.ZERO_INTENSITY) {
-				massSpectrum.addIon(new Ion(ion, intensity));
+				massSpectrum.addIon(new Ion(ion, intensity), false);
```

Comment:
That should be ok here as distinct ions are used. Watch out: if the same ion with a different intensity is added, the previous intensity will be replaced.